app/helpers/pouchDB.js

```diff
@@ -79,6 +89,67 @@ runMigration(
   }
 );
 
+// Get PouchDB Invoices (Used for Exporting)
+const pouchDBInvoices = () => {
+  const invoices = new Promise((resolve, reject) => {
+    const invoicesStream = new MemoryStream();
+    let invoicesStreamObject;
+
+    invoicesStream.on('data', chunk => {
+      const json = JSON.parse(chunk.toString());
+      if (json.docs) {
+        invoicesStreamObject = json;
+      }
+    });
+
+    invoicesStream.on('end', () => {
+      resolve(invoicesStreamObject);
+    });
+
+    // Dump invoices to memoryStream
+    invoicesDB.dump(invoicesStream).then(res => {
+      if (res.ok !== true) {
+        reject(new Error('Exporting PouchDB to CSV failed!'));
+      }
+    });
+  });
+
+  return invoices;
+};
+
+// Get PouchDB Contacts (Used for Exporting)
+const pouchDBContacts = () => {
```

Comment:
Same as above, why don't we use [`getAllDocs`](https://github.com/hql287/Manta/blob/dev/app/helpers/pouchDB.js#L101-L115) instead?